proprietary/tokens/src/components/ams/blockquote.tokens.json

```diff
@@ -6,7 +6,11 @@
       "font-size": { "value": "{ams.text.level.3.font-size}" },
       "font-weight": { "value": "{ams.text.font-weight.bold}" },
       "line-height": { "value": "{ams.text.level.3.line-height}" },
-      "inverse-color": { "value": "{ams.color.primary-white}" }
+      "inverse": {
+        "color": {
```

Comment:
This means the modifier name is now 'inverse', right, not 'inverse-color'?

So it should be `ams-paragraph--inverse`, for example?

Or should the token be `ams.paragraph.inverse-color.color`?

Reply:
Good point. I’ve mentioned this change in [a broader ticket](https://gemeente-amsterdam.atlassian.net/jira/software/c/projects/DES/boards/1378/backlog?epics=visible&selectedIssue=DES-686&text=inverse) on this API. We can leave this as is until then.